packages/wine.rb

```diff
@@ -5,28 +5,44 @@ class Wine < Package
   description 'Wine (originally an acronym for "Wine Is Not an Emulator") is a compatibility layer capable of running Windows applications on several POSIX-compliant operating systems, such as Linux, macOS, & BSD.'
   homepage 'https://www.winehq.org/'
   compatibility 'all'
-  version '3.13'
-  source_url 'https://dl.winehq.org/wine/source/3.x/wine-3.13.tar.xz'
-  source_sha256 '0785d74852232c1f6e07ac081875e92219a4428ddcc8931632fcf1924941bbd4'
-
-  binary_url ({
-  })
-  binary_sha256 ({
-  })
+  version '5.0.2-stable'
+  source_url 'https://dl.winehq.org/wine/source/5.0/wine-5.0.2.tar.xz'
+  source_sha256 'c2c284f470874b35228327c3972bc29c3a9d8d98abd71dbf81c288b8642becbc'
 
   depends_on 'bison'
-  depends_on 'sommelier'
   depends_on 'flex'
+  depends_on 'harfbuzz'
   depends_on 'freetype'
+  depends_on 'glib'
+  depends_on 'zlibpkg'
+  depends_on 'ncurses'
+  depends_on 'libgphoto'
+  depends_on 'libxext'
+  depends_on 'eudev'
+  depends_on 'libxml2'
+  depends_on 'gstreamer'
+  depends_on 'lcms'
+  depends_on 'openldap'
+  depends_on 'libpcap'
+  depends_on 'apulse'
+  depends_on 'pulseaudio'
+  depends_on 'mpg123'
+  depends_on 'sommelier'
+  depends_on 'ffmpeg'
```

Comment:
Please make sommelier the last dependency since it has postinstall instructions we don't want the user to miss.

Reply:
bison, flex, zlibpkg, ncurses and libxml2 are all core dependencies that can be excluded.  For future reference, compare dependencies to https://github.com/skycocker/chromebrew/blob/master/tools/core_packages.txt while listing them out.

Reply:
Done